Sources/StreamChat/StateLayer/MessageState.swift

```diff
@@ -7,31 +7,26 @@ import Foundation
 /// Represents a ``ChatMessage`` and its state.
 @available(iOS 13.0, *)
 public final class MessageState: ObservableObject {
-    private let chat: Chat
     private let messageOrder: MessageOrdering
     private let observer: Observer
 
     let replyPaginationHandler: MessagesPaginationStateHandling
     
-    init(message: ChatMessage, chat: Chat, messageOrder: MessageOrdering, database: DatabaseContainer, clientConfig: ChatClientConfig, replyPaginationHandler: MessagesPaginationStateHandling) {
-        self.chat = chat
+    init(message: ChatMessage, messageOrder: MessageOrdering, database: DatabaseContainer, clientConfig: ChatClientConfig, replyPaginationHandler: MessagesPaginationStateHandling) {
```

Comment:
The linter should be complaining on this formatting, I think

Reply:
Done